ShelbyFinancials/ReadMe.md

```diff
@@ -40,28 +40,32 @@ After install, the Shelby Financials Batch to Journal block was added to your Ba
+| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">4</span> | **Journal Description Lava** Lava for the journal description column per line. Default: Batch.Id: Batch.Name |
+| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">5</span> | **Enable Debug** Outputs the object graph to help create your Lava syntax |
 
 #### Shelby Financials Batches to Journal Block
 
 ![](../.screenshots/ShelbyFinancials/BatchesToJournalProperties.png)
 
+<div style="page-break-after: always;"></div>
+
 | | |
 | --- | ---- |
-| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">1</span><span> | **Name** Block name</span> |
-| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">2</span> | **Detail Page** Link to the Financial Batch Details page |
-| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">3</span> | **Button Text** Customize the text for the export button |
-| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">4</span> | **Months Back** Number of months back that batches should be loaded. This is helpful to prevent database timeouts if there are years of historical batches |
+| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">1</span> | **Detail Page** Link to the Financial Batch Details page |
+| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">2</span> | **Button Text** Customize the text for the export button |
+| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">3</span> | **Months Back** Number of months back that batches should be loaded. This is helpful to prevent database timeouts if there are years of historical batches |
+| <span style="width: 3em; height: 3em; line-height: 3em; background: #d21919; border-radius: 100%; color: white; text-align: center; display: inline-block;">4</span> | **GL Account Grouping** Determines if debit and/or credit lines should be grouped and summed by GL account in the export file |
```

Comment:
Add detailed note describing what fields are grouped by for debit grouping, credit grouping, and financial account grouping